packages/auth/src/authClients/index.ts

```diff
@@ -46,6 +46,7 @@ export interface AuthClient {
   restoreAuthState?(): void | Promise<any>
   login(options?: any): Promise<any>
   logout(options?: any): void | Promise<void>
+  signup(options?: any): void | Promise<void> | Promise<any>
```

Comment:
```suggestion
  signup(options?: any): void | Promise<any>
```

Reply:
Of course the any case covers void as well